charts/tekton-apps/Chart.yaml

```diff
@@ -605,6 +606,82 @@ description: |
           - CreateNamespace=true
     ```
 
+    If you want to enable ignoring deployment replicas count differences in ArgoCD application of your component add `apps[PROJECT].components[NAME].ignoreDeploymentReplicasDiff: true` flag like in the below example (it may be needed for `staging` and `prod` envs, where you have horizontal pod autoscheduling - HPA):
+
+    ```yaml
+    apiVersion: argoproj.io/v1alpha1
+    kind: Application
+    metadata:
+      name: tekton-apps
+      namespace: argo-cd
+      finalizers:
+      - resources-finalizer.argocd.argoproj.io
+      annotations:
+        argocd.argoproj.io/sync-options: SkipDryRunOnMissingResource=true
+        argocd.argoproj.io/sync-wave: "41"
+    spec:
+      destination:
+        server: https://kubernetes.default.svc
+        namespace: ci
+      project: default
+      source:
+        chart: saritasa-tekton-apps
+        helm:
+          values: |
+            environment: staging
+            ...
+            apps:
+              - project: xxx-dev
+                environment: dev
+                enabled: true
+                argocd:
+                  labels:
+                    created-by: xxx
+                    ops-main: xxx
+                    ops-secondary: xxx
+                    pm: xxx
+                    tm: xxx
+                  namespace: xxx-dev
+                  extraDestinationNamespaces:
+                    - argo-cd
+                mailList: xxx@saritasa.com
+                devopsMailList: devops+xxx@saritasa.com
+                jiraURL: https://saritasa.atlassian.net/browse/xxx
+                tektonURL: https://tekton.saritasa.rocks/#/namespaces/ci/pipelineruns
+                slack: client-xxx-ci
+                kubernetesRepository:
+                  name: xxx-kubernetes-aws
+                  branch: main
+                  url: git@github.com:saritasa-nest/xxx-kubernetes-aws.git
+
+                components:
+                  - name: backend
+                    repository: xxx-backend
+                    pipeline: buildpack-django-build-pipeline
+                    applicationURL: https://xxx.site.url
+                    ignoreDeploymentReplicasDiff: true
```

Comment:
since this belongs to argo - let's place it in the argo block:

```
apps:
....
     components:
       - name: backend
         repository: ccf-ccl-backend
         pipeline: buildpack-dotnet-build-pipeline
         applicationURL: https://api.tccfapi.io
         argocd:
           syncWave: 210
           ignoreDeploymentReplicasDiff: true  <-----------HERE
         eventlistener:
           template: buildpack-dotnet-build-pipeline-trigger-template
         triggerBinding:
           - name: docker_registry_repository
             value: 407274476768.dkr.ecr.us-east-1.amazonaws.com/tccfapi/prod/backend
           - name: buildpack_builder_image
             value: 407274476768.dkr.ecr.us-east-1.amazonaws.com/tccfapi/prod/buildpacks/paketo/builder:0.1.340-full
           - name: buildpack_runner_image
             value: 407274476768.dkr.ecr.us-east-1.amazonaws.com/tccfapi/prod/buildpacks/paketo/runner:1.3.39-full-cnb
           - name: kubernetes_branch
             value: feature/upgrade-prod-2022-05-28
```

Reply:
done